src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -61,6 +67,12 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
             await installOasdiffTask;
 
             var generateOpenApiDocsPath = (await swaggerManager.RunSwaggerAsync(this.OpenApiSwaggerDocumentNames, cancellationToken)).ToList();
+            
+            if (!this.CheckIfBaseSpecExists())
```

Comment:
Can this check be done even earlier so we early exit before doing any work or are there certain actions we need to take first before we can do the check?

Reply:
Well, I think we should still generate spec(s) for devs from code even if they don't have base spec. They can use our generated spec as a baseline for thei "base" spec.